src/sagemaker/inputs.py

```diff
@@ -165,6 +165,58 @@ class TransformInput(object):
     """Create a class containing all the parameters.
 
     It can be used when calling ``sagemaker.transformer.Transformer.transform()``
+
+    Args:
+        data (str): The S3 location of the input data that the model can consume.
+        data_type (str): elastic inference accelerator type.
```

Comment:
Question, 

the arg name is "data_type" but the description is "the type of an elastic inference accelerator", which is not a data type?

Reply:
Actually should it be "The data type for batch transform job"?